apps/project/graphql/queries.py

```diff
@@ -63,6 +65,33 @@ def _get_vector_tile_server_type(enum: VectorTileServerNameEnumWithoutCustom):
     )
 
 
+DEFAULT_TEST_RESPONSE_ERROR_MESSAGE: str = "Something unexpected has occurred. Please contact an admin to fix this issue."
+
+
+@strawberry.type
```

Comment:
We can define types in `apps/<app-name>/graphql/types`

For this we can use https://github.com/mapswipe/mapswipe-backend/blob/develop/apps/project/graphql/types/types.py